### DISCUSSION TITLE
use an RF module instead of the PIR module

### DISCUSSION THREAD

**Discussion by kenpachi82** [jameszah/ESP32-CAM-Video-Telegram#41] [General]
hi everybody,
thank you very much for this project which is by far the best using telegram with ESP32 cam.
I wonder if there is a way to use a 433 or 315 MHz RF module instead of the PIR module. I'm working on a project to take pictures of animals but I want to start taking pictures by sending a command by RF.

**Comment by FBMinis** [comment]
What kind of transmitter will you be using to send the command from?

**Reply by kenpachi82** [reply]
Thank you for your reply dear. 
I have some garage remotes . I triyed to use existant libraries to capture the rf signal but without succes. I can do it with arduino but not for this espcam

**Reply by FBMinis** [reply]
I would use one of the Arduino board output pins to trigger the interrupt on GPIO13 on the ESP32-Cam.

**Reply by kenpachi82** [reply]
It's what i'm thinking about. I have just to adapt pir interrupt part in your code. Am i right?

**Reply by FBMinis** [reply]
> It's what i'm thinking about. I have just to adapt pir interrupt part in your code. Am i right?

If you connect the Arduino board output to GPIO13 instead of the PIR, I don't see the need to change the code.

**Reply by kenpachi82** [reply]
Thank you very much for your help
